doc/EnvironmentVariables.md

```diff
@@ -16,6 +16,7 @@ You can set Edge Agent and Edge Hub environment variables in the Azure portal. I
 | DisableDeviceAnalyticsMetadata       | Whether to disable sending [basic metadata](https://github.com/Azure/azure-iotedge/blob/main/README.md#telemetry) about the device to Microsoft. | bool                         | false                      |
 | EnableK8sServiceCallTracing          | Whether to enable logging for K8s requests that the Agent makes                                                                 | bool                                          | false                      |
 | EnableOrphanedIdentityCleanup        | Whether to enable removal of any orphaned identities<br>NOTE: There is an issue with the `managedBy` tag for module identities that **WILL** cause this logic to remove **Host-Level** modules (i.e. managed by something other than `IotEdge`)</br> | bool | false |
+| EnableSdkDebugLogs                   | If set, emits SDK client events to logs                                                                                         | bool                                          | false                      |
```

Comment:
I wonder if this should include a warning, similar to https://github.com/Azure/azure-iot-sdk-csharp/blob/main/tools/CaptureLogs/readme.md#console-logging: "Note that enabling this setting will substantially slow down execution."

Reply:
Will add